includes/wc-cart-functions.php

```diff
@@ -133,7 +133,7 @@ function wc_add_to_cart_message( $products, $show_qty = false, $return = false )
 	if ( $return ) {
 		return $message;
 	} else {
-		wc_add_notice( $message );
+		wc_add_notice( $message, apply_filters( 'wc_add_to_cart_notice_type', 'success' ) );
```

Comment:
We don't allow `wc_` prefixes for hooks.
Need to update to `woocommerce_add_to_cart_notice_type`.